src/middleware/authMiddleware.js

```diff
@@ -0,0 +1,35 @@
+const formatResponse = require("../helpers/serverResponse");
+const jwt = require("jsonwebtoken");
+const { key } = require("../../config/jwtKey");
+
+exports.allowAccess = function (roles) {
+  return function (req, res, next) {
+    try {
+      const token = req.headers.authorization.split(" ")[1];
+      if (!token) {
+        throw {
```

Comment:
you should use `return res.status(401).send(formatResponse(...))` instead of throw {}